packages/sdk/src/universe/types.ts

```diff
@@ -0,0 +1,91 @@
+export enum AssetClass {
+  ERC20 = "ERC20",
+  ETH = "ETH",
+  ERC721 = "ERC721",
+  ERC1155 = "ERC1155",
+}
+
+export enum OrderSide {
+  BUY,
+  SELL,
+}
+
+export type AssetType = {
+  assetClass: string;
+  data: string;
+};
+
+export type Asset = {
+  assetType: AssetType;
+  value: string;
+};
+
+export type LocalAssetType = {
+  assetClass: string;
+  contract?: string;
+  tokenId?: string;
+};
+
+export type LocalAsset = {
+  assetType: LocalAssetType;
+  value: string;
+};
+
+export type OrderKind = "single-token";
+
+export type Order = {
+  kind?: OrderKind;
+  type: string;
+  side: OrderSide;
```

Comment:
Also, we should add a `getInfo` or similar method as mentioned in a previous comment to extract the order side. Basically we will do something like:
- if makeAsset = ERC20 and takeAsset = ERC721/ERC1155 then side = buy
- if makeAsset = ERC721/ERC1155 and takeAsset = ETH/ERC20 then side = sell
- otherwise, error (we don't support bundles or any other order types yet)